frame/assets/rpc/src/lib.rs

```diff
@@ -6,10 +6,10 @@ use jsonrpc_derive::rpc;
 use sp_api::ProvideRuntimeApi;
 use sp_blockchain::HeaderBackend;
 use sp_runtime::{generic::BlockId, traits::Block as BlockT};
-use sp_std::sync::Arc;
+use sp_std::{sync::Arc, vec::Vec};
 
 #[rpc]
-pub trait AssetsApi<BlockHash, AssetId, AccountId, Balance>
+pub trait AssetsApi<BlockHash, AssetId, AccountId, Balance, Asset>
```

Comment:
Is the generic type necessary? Since we use a concrete type for `Asset`

Reply:
do you mean the AssetsApi<> generic?

Reply:
Yes, the `Assets` generic type

Reply:
Not sure I get what you mean, Asset is a Type required by the generic AssetsApi, can you make it clear

Reply:
youre right, i have pushed a new commit that removes it